docs/cs_v6_release_guide.md

```diff
@@ -447,6 +447,36 @@ sed -i "s|detail->sysz|detail->systemz|g" $1
 
 Write it into `rename.sh` and run it on files with `sh rename.sh <src-file>`
 
+#### Python binding compatibility
+
+The Python bindings were changed to match the renamed architectures. To allow using the old `ARM64_*` and `SYSZ_*` constants in scripts running against Capstone v6+, compatibility modules can be imported, which monkey-patch the modules to export the same constants with the old name as well. The compatibility module has to be imported **before** anything else from capstone.
+
+To load the ARM64 constants, which were moved to the `capstone.aarch64` module, import the old `capstone.arm64` module first explicitly:
```

Comment:
```suggestion
To load the ARM64 constants import the old `capstone.arm64` module first:
```
Just a suggestion, but you can add the "ARM64 was moved to AArch64" information to the top of the chapter. This way we don't mix up two different topics in a single sentence.